### ISSUE TITLE
How to force Web Audio Api or at least detect fallback to HTML5?

### ISSUE THREAD

**Issue by chadananda** [goldfire/howler.js#1278]
I'm cutting my audio up into sprites for precision manipulation in a way that will not work at all with HTML5. I need to force Web Audio Api only. I've noticed on some devices (android w/ chrome) that Howler seems to slip back to HTML5, mangling playback completely.

I'm not sure why Howler seems to be falling back to HTML5 sometimes but I need it to stop. How can I detect when this happens and then why?

**Comment by chadananda** [comment]
Found `howl._webAudio` which is a big help.

**Comment by chadananda** [comment]
It would be helpful to throw some console error when falling back to html5, preferably with a reason so it can be fixed.

**Comment by chadananda** [comment]
Seems like Howler starts out with Web Audio API then switches right back to HTML5 the very first time any network glitch comes along -- basically killing any app that cannot use HTML5. And there seems no way to switch it back.

**Comment by chadananda** [comment]
It would be great to have a **configuration option to disable HTML5 audio**. 

For some applications HTML5 audio does not work at all, and Howler's eagerness to switch to HTML5 audio at the slightest provocation just breaks the app.

**Comment by shahburhan** [comment]
You actually can set it to use just the Audio Api. 
You need to set the html5 property to false on the howl instance and voila.

**Comment by chadananda** [comment]
That is not correct. `False` is the default value. Howler allows you to exclude Web API with `true` but when it is set to `false` Howler will switch back to `true` if anything at all goes wrong -- instead of throwing an error. It does not seem possible to exclude HTML5 in Howler.

**Comment by chadananda** [comment]
It would be nice to have an option to simply throw an error if something goes wrong -- instead of assuming we can switch hack to HTML5 audio. Is there any work-around?

**Comment by shahburhan** [comment]
That is correct, there should be something like that out of the box. But may be until that's there, we can watch for the switch and when it happens, the audio can be forced off and an error can be flagged.
